internal/k8s/controller.go

```diff
@@ -1861,6 +1878,34 @@ func (lbc *LoadBalancerController) updateVirtualServerRoutesStatusFromEvents() e
 	return nil
 }
 
+func (lbc *LoadBalancerController) updatePolicyStatus() error {
```

Comment:
for consistency with similar `updateVirtualServersStatusFromEvents`
```suggestion
func (lbc *LoadBalancerController) updatePoliciesStatus() error {
```